### ISSUE TITLE
Missing docs with examples of all different ways of creating bucket classes.

### ISSUE THREAD

**Issue by bipuladh** [noobaa/noobaa-operator#107]
We need to update the docs to show how different tiering combinations can be used. And explain the structure of the YAML to be used. https://github.com/noobaa/noobaa-operator/blob/master/doc/bucket-class-crd.md

**Comment by guymguym** [comment]
Few examples from @bipuladh :
```
# ex1 - single tier with single BS
spec:
  placementPolicy:
    tiers:
    - backingStores: ["bs1"]

# ex2 - two tiers each with single BS
spec:
  placementPolicy:
    tiers:
    - backingStores: ["bs1"]
    - backingStores: ["bs2"]

# ex3 - one tier with Mirror on two BS
spec:
  placementPolicy:
    tiers:
    - placement: Mirror
      backingStores: ["bs1", "bs2"]

# ex4 - two tiers, first with Mirror, and second with Spread
spec:
  placementPolicy:
    tiers:
    - placement: Mirror
      backingStores: ["bs1", "bs2"]
    - placement: Spread
      backingStores: ["bs3", "bs4", "bs5"]
```

**Comment by ron1** [comment]
Docs should also clarify whether or not a single backingStore can appear in more than one placement within a single bucketClass or accross multiple bucketClasses.

**Comment by guymguym** [comment]
@ron1 Good point. The design is that any backingstore name should appear no more than once across the bucket class placementPolicy.

**Comment by ron1** [comment]
@guymguym Can a backingstore name appear in the placementPolicies of two different bucket classes?

**Comment by guymguym** [comment]
@ron1 Yes. BackingStores can be used by multiple BucketClasses and eventually between Buckets.
So in most cases you would just need a BackingStore per cloud/region.

Note that noobaa core uses these backing stores to store encrypted chunks and therefore the sharing is just in terms of storage capacity and not in the data level.

**Comment by guymguym** [comment]
Fixed in #113
